galaxy.yml

```diff
@@ -2,7 +2,7 @@
 namespace: kubernetes_sigs
 description: Deploy a production ready Kubernetes cluster
 name: kubespray
-version: 2.25.0
+version: 2.25.2
```

Comment:
Should be 2.25.1 (next version on this branch)

Reply:
I followed step 9 from release process for the release of 2.25.1 (X.Y.Z):
> (For minor releases) On the release-X.Y branch: bump the version in galaxy.yml to the next expected minor release (X.Y.z with z = Z + 1), make a Pull Request.

Reply:
Created an additional PR #11635